src/land/ghcn_snod2ioda.py

```diff
@@ -197,12 +212,15 @@ def main():
 
     args = parser.parse_args()
 
-    writer = iconv.NcWriter(args.output, locationKeyList)
+    # Read in the GHCN snow depth data
+    snod = ghcn(args.input, args.fixfile, args.date, args.mask)
 
-    # Read in the profiles
-    snod = ghcn(args.input, args.fixfile, args.date, args.mask, writer)
+    # setup the IODA writer
+    writer = iconv.IodaWriter(args.output, locationKeyList, DimDict)
 
-    writer.BuildNetcdf(snod.outdata, snod.loc_mdata, snod.var_mdata, AttrData, snod.units)
+    snod.varAttrs[('altitude', 'MetaData')]['units'] = 'm'
```

Comment:
Similar to the other PR, I advise making the units assignment in the main class (perhaps around line180) and not doing it here.

Reply:
Yes, moved as suggested.